js/app/cart/DatePicker.jsx

```diff
@@ -102,7 +103,7 @@ class DatePicker extends Component {
             {
               dates.map(date => (
                 <option key={ date } value={ date } >
-                  { moment(date).format('dddd DD MMM') }
+                  { moment(date).calendar(null, { sameDay: '[Aujourd\'hui]',  nextDay: '[Demain]', sameElse: 'dddd DD MMM'}) }
```

Comment:
Can we use `window.__i18n`?

Reply:
I think we should use a pure-JS solution (like [this](https://github.com/martinandert/counterpart) (didnt look into it) to translates our React component, `window.__i18n` is nice but don't scale.
I opened #99 to discuss this.